subsys/net/lib/zperf/zperf_shell.c

```diff
@@ -76,9 +76,9 @@ static struct in_addr shell_ipv4;
 
```

Comment:
Looks ok, please change the commit subject to `net: zperf: ....` as the net is the subsystem so there is no need to have subsys prefix there.

Reply:
Another option (if we want to keep using 1024), is to change the printouts to `Kibit/s` or `Mibit/s` etc, this way the values would not change and could be compared with the earlier results.

Reply:
Hi jukkar,

It will be confusing when common use is Kbps(like iperf) but zperf uses Kibits/s.

Reply:
> And it will be confusing when common use is Kbps(like iperf) but zperf uses Kibits/s.

Yes, I agree, it would be nice if we can compare the iperf numbers.

I think we could add a note to migration guide ` doc/releases/migration-guide-3.6.rst` about this so that it is clear that the performance values change.  Could you add such a commit?

Reply:
Added commit is this PR